packages/api/src/routes/farmExpenseTypeRoute.js

```diff
@@ -43,4 +43,11 @@ router.delete(
   farmExpenseTypeController.delFarmExpenseType(),
 );
 
+router.put(
+  '/:expense_type_id',
+  hasFarmAccess({ params: 'expense_type_id' }),
```

Comment:
I think this should be farm_id otherwise you are able to change the farm_id using the body param to another farm (that you may nat have access to). I think you can also have multiple hasfarmAccess calls if this one is needed.